addons/website_sale/controllers/main.py

```diff
@@ -1547,15 +1546,6 @@ def _create_new_address(
             creation_context
         ).create(address_values)
 
-    def _get_extra_billing_info_route(self, order_sudo):
```

Comment:
If we remove this hook completely, should not we remove all overrides as well?

Reply:
This is done in the related enterprise PR.
BTW, could you help me get it approved ?

Reply:
Indeed, thank you, I did not notice!
Sure, I pinged @chevalierv .